### DISCUSSION TITLE
Can't login on Google account anymore

### DISCUSSION THREAD

**Discussion by MBilderbeek** [kewisch/gdata-provider#396] [Support]
Since about a week (maybe 2) or so, I cannot login anymore on my Google account. My normal password isn't working, also an app password isn't working. The login dialog is just telling me it's wrong.
Before, I have used it for years without any issues (and didn't have to provide the password after the first time).

So, something changed... could be the 24 January release?

What's going on?

**Comment by kewisch** [comment]
I believe this was around the time I had the rate limiting errors. Should be fixed by now :)
